### DISCUSSION TITLE
There is a system-cursor and osu-cursor on the game screen when mouse is replugged

### DISCUSSION THREAD

**Discussion by labo-China** [ppy/osu#14844] [Q&A] [answered]
As title, this problem occurs definitely. The cursor in game is still works, but Windows cursor does not work in game.
And because of the different DPI between system and game, two cursors will move by different speed.
I have been changed my mouse, but the problem always occurs.

Platform: Windows 10 19042.1237
Mouse: A4Tech OP-520NU
OSU: 20210920
Raw-input: enabled

**Comment by PercyDan54** [comment, marked as answer]
From the version I doubt it's a stable issue.
https://github.com/ppy/osu-stable-issues or https://osu.ppy.sh/community/forums/5
